appveyor.yml

```diff
@@ -0,0 +1,19 @@
+install:
+  - cinst -y jdk7
```

Comment:
That's a bummer, but I'd prefer if we kept to java8 since that's what we are testing elsewhere (and there are differences).